decidim-proposals/spec/lib/decidim/proposals/component_spec.rb

```diff
@@ -285,5 +285,19 @@
         expect(subject).to match_array([unassigned_proposal, assigned_proposal])
       end
     end
+
+    context "when proposal is moderated" do
+      let(:hidden_proposal) { create :proposal, component: }
+      let!(:moderation) { create(:moderation, hidden_at: 6.hours.ago, reportable: hidden_proposal) }
+      let!(:user) { create :user, admin: true, organization: }
+
+      it "exports all proposals from the component" do
+        expect(subject).to include(unassigned_proposal, assigned_proposal)
+      end
+
+      it "excludes hidden comments" do
```

Comment:
We should change this string 
```suggestion
      it "excludes the hidden proposals" do
```